skbuild/cmaker.py

```diff
@@ -133,21 +133,25 @@ def __init__(self, cmake_executable=CMAKE_DEFAULT_EXECUTABLE):
         self.cmake_version = get_cmake_version(self.cmake_executable)
         self.platform = get_platform()
 
-    def get_cached_generator_name(self):
-        """Reads and returns the cached generator from the :func:`skbuild.constants.CMAKE_BUILD_DIR()`:.
-        Returns None if not found.
-        """
+    def get_cached(self, item):
+        """Return the cached value if possible, otherwise return None"""
```

Comment:
```suggestion
        """If possible, return the cached value from the :func:`skbuild.constants.CMAKE_BUILD_DIR()`, otherwise return None"""
```

Reply:
I changed `If possible` by `If set`

Reply:
Thanks! Working on getting a test working int this. Not trivial to get the local scikit-build into the PEP 518 requirements.